lib/barnes.rb

```diff
@@ -54,7 +54,12 @@ def self.start(interval: DEFAULT_INTERVAL, aggregation_period: DEFAULT_AGGREGATI
         panels << Barnes::ResourceUsage.new(sample_rate)
       end
 
-      Periodic.new reporter: reporter, sample_rate: sample_rate, panels: panels
+      Periodic.new(
+        reporter:    reporter,
+        sample_rate: sample_rate,
+        panels:      panels,
+        debug:       ENV['BARNES_DEBUG']
```

Comment:
I'm sure what you have is conventional ruby, but would it make sense to write this as `not ["", "false"].include?(ENV['BARNES_DEBUG'])`

As it stands, it's confusing to set BARNES_DEBUG=false and have debug actually on.

Reply:
It's fairly conventional since "false" in ruby is also truthy. i.e. it's just as confusing to set some `application.configuration.option = "false"` and have it still be on, though it's correct. 

Generally would set it to anything to enable and then unset to disable. You see this in Rails:

```
  if ENV["RAILS_LOG_TO_STDOUT"].present?
    logger           = ActiveSupport::Logger.new(STDOUT)
    logger.formatter = config.log_formatter
    config.logger    = ActiveSupport::TaggedLogging.new(logger)
  end
```

I don't really intend on using this, other than for the tests.

Reply:
do you mean `"false"` or `false` is truthy? Environment variables, of course, aren't ruby code so are treated as strings, all of which are truthy, including the empty string, right?

Anyway, if it's conventional ruby to do `ENV['foo'].present?` then so be it, but it's odd to me. :)

Reply:
fwiw, in rails/spring

```ruby
disable = ENV["DISABLE_SPRING"]
enabled = (disable.nil? || disable.empty? || disable == "0")
```

(I have DISABLE_SPRING=1 in my bash profile ;)

Reply:
In general in Rails we check for presence https://github.com/rails/rails/pull/18100#issuecomment-67704864

There's really no great answer because no matter which side you choose you will violate people's assumptions.